.github/workflows/unit-tests-workflow.yml

```diff
@@ -39,10 +39,17 @@ jobs:
         with:
           path: kibana/plugins/anomaly-detection-kibana-plugin
       - name: Bootstrap the plugin
+        continue-on-error: true
+        run: |
+          cd kibana/plugins/anomaly-detection-kibana-plugin
+          yarn kbn bootstrap
```

Comment:
this step looks like to be attempting bootstrap, maybe you can change the name to "Attempt bootstrap"

Reply:
sure, yeah